site/content/en/docs/Reference/fleet.md

```diff
@@ -83,6 +85,27 @@ spec:
         logLevel: Info
         grpcPort: 9357
         httpPort: 9358
+      #
+      # [Stage:Beta]
+      # [FeatureFlag:CountsAndLists]
+      # Counts and Lists provides the configuration for generic (player, room, session, etc.) tracking features.
+      # Now in Beta, and enabled by default
+      # counters:
+      #   players:
+      #     count: 0 # Initial Value
+      #     capacity: 10
+      #   sessions:
+      #     count:  # Count and/or capacity must be listed (but may be nil) otherwise the counter will by dropped by the CRD schema.
+      # lists:
+      #   players:
+      #     capacity:  # Capacity and/or values must be listed (but may be nil) otherwise the list will be dropped by the CRD schema.
```

Comment:
```suggestion
      #     capacity:  100
```

Let's just give it a value.

Reply:
I have now removed the verbose comments and made it match landing page with:

```yaml
        players:
          values: []
```